prime-router/src/main/kotlin/tokens/OktaAuthentication.kt

```diff
@@ -2,53 +2,122 @@ package gov.cdc.prime.router.tokens
 
```

Comment:
This class now integrates and refactors functionality that used to be confusingly spread across OktaAuthentication.kt and Authentication.kt.  The latter is now removed!